### PR TITLE
Add commands from the demo gif to the README.md as text

### PR DESCRIPTION
**Pull request by anze3db**
Can we add the commands from the gif in text form so that they are easy to copy/paste? This will make it easier to try out the project locally.

### CODE REVIEW

**Comment by danielhoherd** [comment]
god yes. the first thing i noticed when i loaded up this project was how terrible the experience of looking at the example was. just show me the full terminal buffer.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ PyApp is a wrapper for Python applications that bootstrap themselves at runtime.
 
 <img src="https://raw.githubusercontent.com/ofek/pyapp/master/docs/assets/images/example.gif" alt="PyApp example workflow" role="img">
 
+```console
+export PYAPP_PROJECT_NAME=cowsay
+export PYAPP_PROJECT_VERSION=6.0
+cargo install pyapp --quiet --root out
+mv out/bin/pyapp cowsay && chmod +x cowsay
+./cowsay -t 'Hello, World!'
+```
```
Comment (ofek) [inline README.md:22]: See the [how-to](https://ofek.dev/pyapp/latest/how-to/) for a detailed example walkthrough.

Comment (ofek) [inline README.md:22]:
```suggestion
See the [how-to](https://ofek.dev/pyapp/latest/how-to/) for a detailed example walkthrough.
```

**Review by ofek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ofek) [reply README.md:22]: Documentation coming later tonight!

**Review by ofek — APPROVED** [review]
Thank you!

**Comment by ofek** [comment]
https://ofek.dev/pyapp/latest/how-to/

**Pull request merged**